src/main/java/life/qbic/App.java

```diff
@@ -35,30 +35,36 @@ private static Boolean isNotNullOrEmpty(String envVariableCommandLineParameter)
   }
 
   /**
-   * Logs into OpenBIS asks for and verifies password.
+   * Logs into OpenBIS, asks for and verifies password.
    *
    * @return An instance of the Authentication class.
    */
-  public static Authentication loginToOpenBIS(
-      char[] password, String user, String as_url) {
+  public static OpenBIS loginToOpenBIS(
+      char[] password, String user, String url) {
+    //setupLog();
 
-    // Ensure 'logs' folder is created
-    File logFolder = new File(Configuration.LOG_PATH.toAbsolutePath().toString());
-    if (!logFolder.exists()) {
-      boolean logFolderCreated = logFolder.mkdirs();
-      if (!logFolderCreated) {
-        LOG.error("Could not create log folder '" + logFolder.getAbsolutePath() + "'");
-        System.exit(1);
-      }
-    }
+    OpenBIS authentication = new OpenBIS(url);
+
+    return tryLogin(authentication, user, password);
+  }
+
+  /**
+   * Logs into OpenBIS, asks for and verifies password, includes Datastore Server connection.
+   *
+   * @return An instance of the Authentication class.
+   */
+  public static OpenBIS loginToOpenBIS(
+      char[] password, String user, String url, String dssUrl) {
+    //setupLog();
```

Comment:
```suggestion
    //setupLog();
```
Why is this commented out? Is this not necessary anymore 🤔